tests/NLog.UnitTests/LogFactoryTests.cs

```diff
@@ -48,17 +48,19 @@ public void Flush_DoNotThrowExceptionsAndTimeout_DoesNotThrow()
         {
             using (new NoThrowNLogExceptions())
             {
-                LogManager.ThrowExceptions = true;
-
-                LogManager.Configuration = XmlLoggingConfiguration.CreateFromXmlString(@"
+                var config = XmlLoggingConfiguration.CreateFromXmlString(@"
                 <nlog throwExceptions='false'>
                     <targets><target type='MethodCall' name='test' methodName='Throws' className='NLog.UnitTests.LogFactoryTests, NLog.UnitTests.netfx40' /></targets>
                     <rules>
                         <logger name='*' minlevel='Debug' writeto='test'></logger>
                     </rules>
                 </nlog>");
 
-                ILogger logger = LogManager.GetCurrentClassLogger();
+                var LogFactory = new LogFactory(config);
+
+                LogFactory.ThrowExceptions = true;
```

Comment:
`LogManager.ThrowExceptions = true` has very special meaning when unit-testing. Why the change to `LogFactory.ThrowExceptions = true`  and enabling after the config-load?

Reply:
good one, I prefer also undo of this change